### ISSUE TITLE
PyPI version badge says "Maximum response size exceeded"

### ISSUE THREAD

**Issue by kroitor** [badges/shields#6422]
Hi!

We are having an issue with this PyPI version shieldbadge rendering wrong here: https://github.com/ccxt/ccxt#ccxt--cryptocurrency-exchange-trading-library

<img width="1049" alt="Screenshot 2021-04-23 at 18 58 03" src="https://user-images.githubusercontent.com/1294454/115898159-ea085980-a465-11eb-93b8-f7bb23b2ffea.png">

-------------------------------------

Here it is live: 

[![PyPI](https://img.shields.io/pypi/v/ccxt.svg)](https://pypi.python.org/pypi/ccxt)

It was working fine for a long while, then stopped displaying the version suddenly. 

It would be great if anyone shed any light on how to fix this. 

Thank you!

**Comment by chris48s** [comment]
Hello. The message means pretty much what it says. We set a limit on the size of response we will accept as input. If we hit that limit, we stop the request. This prevents users from using the dynamic or endpoint badges to DoS us by giving us some arbitrarily large API response. At the moment we set that limit at 10Mb: https://github.com/badges/shields/blob/3e8cdf44be772b71cd5398e1336d270b8e89d994/config/default.yml#L34 Parsing 10Mb of JSON to write out an SVG with a single string on it seems a pretty reasonable upper bound.
Long story short, https://pypi.org/pypi/ccxt/json serves up more than 10Mb of JSON, so we fail.
```
$ curl --silent "https://pypi.org/pypi/ccxt/json" -w 'size: %{size_download}\n' -o /dev/null | sed 's/[^0-9]//g' | numfmt --to=iec
11M
```
so that's why it failing. This is an interesting case because I think this is the first time I've seen one of the upstream APIs we consume (as opposed to a user supplied input) exceed this limit.

**Comment by kroitor** [comment]
@chris48s thanks for your explanation! We will try to reduce the size of that JSON.
